### PR TITLE
Add AWS DynamoDB Streams Scaler docs

### PR DESCRIPTION
**Pull request by yokawasa**
Signed-off-by: Yoichi Kawasaki <yokawasa@gmail.com>

_Provide a description of what has been changed_

I add a new docs for AS DynamoDB Streams Scaler

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

**Review by JorTurFer — APPROVED** [review]
LGTM! Thanks for this! ❤️

File: content/docs/2.8/scalers/aws-dynamodb-stream.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+go_file = "aws_dynamodb_stream_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `aws-dynamodb-stream` trigger that scales based on the shard count of AWS DynamoDB Streams.
+
+```yaml
+triggers:
+- type: aws-dynamodb-stream
```
Comment (tomkerkhove) [inline content/docs/2.8/scalers/aws-dynamodb-stream.md:16]: Big nit - The scaler is called `AWS DynamoDB Streams` but our type is `aws-dynamodb-stream`.
Should we align them?

Comment (yokawasa) [reply content/docs/2.8/scalers/aws-dynamodb-stream.md:16]: @tomkerkhove 
let me align them to `aws-dynamodb-streams` and `AWS DynamoDB Streams` as the official service name looks DynamoDB Streams ([ref](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html)). I'll modify shortly. thank you!

Comment (JorTurFer) [reply content/docs/2.8/scalers/aws-dynamodb-stream.md]: remember to update also this docs 🙏

**Review by JorTurFer — APPROVED** [review]

**Review by zroubalik — APPROVED** [review]

**Pull request merged**